themes/socialblue/templates/profile/profile--profile--statistic--sky.html.twig

```diff
@@ -0,0 +1,51 @@
+{{ attach_library('socialbase/hero') }}
```

Comment:
The same questions about making this more generic exist as for `themes/socialblue/templates/group/group--statistic--sky.html.twig`.